Tribler/Main/vwxGUI/list_details.py

```diff
@@ -522,20 +525,23 @@ def updateDetailsTab(self):
         self.channel.Show(show_channel)
 
         # Toggle thumbnails
-        # FIXME(lipu): fix the thumbnail path to use metadata
-        thumb_dir = os.path.join(u"",
-                                 binascii.hexlify(self.torrent.infohash))
-        thumb_files = [os.path.join(dp, fn) for dp, _, fns in os.walk(thumb_dir)
-                       for fn in fns if os.path.splitext(fn)[1] in THUMBNAIL_FILETYPES]
-        show_thumbnails = bool(thumb_files)
+        if self.torrent.metadata and 'thumb_hash' in self.torrent.metadata:
```

Comment:
@NielsZeilemaker I applied the `metadata=False` method and did a few tests with two instances again. So, when I could see two thumbnail torrents on the front page (which always reloads the thumbnail torrents), I switched to my subscribed channel and I could only see one torrent with thumbnail. The other one didn't reload its metadata_dict.